src/azul/indexer/transform.py

```diff
@@ -68,3 +71,11 @@ def get_aggregator(cls, entity_type) -> EntityAggregator:
         entities of types other than X.
         """
         raise NotImplementedError
+
+    @classmethod
```

Comment:
Does not need to be a class method. Remember to only use class methods when it is impossible or prohibitively expensive to obtain an instance.

Reply:
I believe in this case it needs to be class method as in `IndexService._aggregate()` I only have the transformer class (obtained from `DocumentService.transfomers()`) and no bundle that could be used to create the transformer object.

Reply:
You are right.